workspaces/tasks.py

```diff
@@ -37,25 +37,28 @@ def send_invitation_email(invitation_pk: int):
 
 
 @app.task
-def send_added_to_workspace_email(workspace_id: int, user_id: int):
-    from routers.account import members_route
-    from workspaces.models import Workspace
+def send_added_to_workspace_email(invite_id: int, user_id: int):
```

Comment:
please update all usages of send_added_to_workspace_email

Reply:
ah, good catch!
i had earlier removed the signal handler entirely where this was used.

Reply:
fixed with `WorkspaceInvite.objects.create_and_send_invite` - it takes care of whether to auto accept or send an email